src/procedures/proc_fulltext_create_index.c

```diff
@@ -42,29 +43,15 @@ static ProcedureResult _validateIndexConfigMap
 		return PROCEDURE_ERR;
 	}
 
-	if(multi_config) {
-		GraphContext *gc = QueryCtx_GetGraphCtx();
-		Index idx = GraphContext_GetIndex(gc, label.stringval, NULL, 0,
-				IDX_FULLTEXT, SCHEMA_NODE);
-		if(idx != NULL) {
-			ErrorCtx_SetError(EMSG_INDEX_ALREADY_EXISTS);
-			return PROCEDURE_ERR;
-		}
-	}
-
 	//--------------------------------------------------------------------------
 	// validate stopwords
 	//--------------------------------------------------------------------------
 
 	if(stopword_exists) {
 		if(SI_TYPE(sw) == T_ARRAY) {
-			uint stopwords_count = SIArray_Length(sw);
-			for (uint i = 0; i < stopwords_count; i++) {
-				SIValue stopword = SIArray_Get(sw, i);
-				if(SI_TYPE(stopword) != T_STRING) {
-					ErrorCtx_SetError(EMSG_MUST_BE, "Stopword", "string");
-					return PROCEDURE_ERR;
-				}
+			if(!SIArray_AllOfType(sw, T_STRING)) {
```

Comment:
isn't this function now same as in index_fulltext_create.c?

Reply:
Similar, this one knows how to handle all of the different configuration one can specify when creating a full-text index with multiple fields, where each field has its own configuration, this is not possible with the `CREATE FULLTEXT INDEX` syntax